bot/src/main/kotlin/io/heapy/kotbot/bot/rule/Actions.kt

```diff
@@ -14,3 +16,7 @@ data class KickUserAction(
     val chatId: Long,
     val userId: Int
 ) : Action()
+
+@Suppress("FunctionName")
```

Comment:
Using builder-function name similarly to those in std-lib.

There are other options, but I suppose they are not good:

1. Secondary constructor (bloating data-class body).
2. Companion object + extension to companion object (better, but not as clear as builder function).

Reply:
I rather go with option 2, or add extension to `Update` to convert it to different actions

Reply:
> I rather go with option 2

May I ask why?

> or add extension to Update to convert it to different actions

I guess extension-fun semantics isn't very clear in this case. :)

Reply:
> May I ask why?

I don't like any additional `Suppress` annotations in code, until there are some good reasons to do this. For example, I suppress "nothing to inline" in high-loaded server code for small extensions that help with readability, and shouldn't add any additional calls. But in case of factory function there are alternative paths exists.

> I guess extension-fun semantics isn't very clear in this case. :)

I found `return listOf(update.kickUser(), update.deleteMessage())` pretty self-explanatory and clean.